docker/scheduler/package.json

```diff
@@ -7,8 +7,7 @@
     },
     "scripts": {
       "scheduler:batch": "node dist/scheduler/batch.js",
-      "scheduler:annual-review-worker": "node dist/scheduler/annualReviewWorker.js",
```

Comment:
Out of curiosity what's the reason for removing the worker? Or the script for the worker?

Reply:
it's been replaced with scheduler:worker so it's still needed